arch/arm64/configs/rockchip_linux_defconfig

```diff
@@ -72,7 +72,7 @@ CONFIG_SWP_EMULATION=y
 CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
 # CONFIG_EFI is not set
-CONFIG_BINFMT_MISC=m
```

Comment:
So, it is enabled. It does not seem that we need to do anything, you just have to load `binfmt_misc` kernel module.